### DISCUSSION TITLE
Enable/disable charge/discharge on battery-level

### DISCUSSION THREAD

**Discussion by cm2962** [Louisvdw/dbus-serialbattery#404] [Ideas]
Hi Louis,
just wondering if there is any way to enable/disable charging for single batteries (when multiple in use).
I see that modbus provides registers for that. But changes seem to be rejected.

Cheers
Christian

**Comment by Louisvdw** [comment]
There is no easy way to do that no. Each BMS is different and will have to be applied. The driver at the moment only reads data from the BMS and this will require to set data. #31 and #40 are similar requests

**Reply by cm2962** [reply]
Okay that sounds reasonable. Even if it would be great :)

**Comment by cm2962** [comment]
Ok, that sounds reasonable. Thank you
